src/themes/default/globals/colors.less

```diff
@@ -3,617 +3,617 @@
-    shadow         : @brownTextShadow;
-    lightShadow    : @lightBrownTextShadow;
-    hover          : @brownHover;
-    lightHover     : @lightBrownHover;
-    ribbon         : @brownRibbonShadow;
-    invertedRibbon : @brownInvertedRibbonShadow;
-    tertiary       : @brownTertiaryColor;
-    tertiaryHover  : @brownTertiaryColorHover;
-    tertiaryFocus  : @brownTertiaryColorFocus;
-    tertiaryActive : @brownTertiaryColorActive;
-    bright         : @brownBright;
-    brightHover    : @brownBrightHover;
-    isDark         : false;
-    isVeryDark     : false;
-  };
-  @grey: {
-    color          : @grey;
-    light          : @lightGrey;
-    border         : @greyBorderColor;
-    background     : @greyBackground;
-    header         : @greyHeaderColor;
-    boxShadow      : @greyBoxShadow;
-    boxFloatShadow : @greyBoxFloatingShadow;
-    text           : @greyTextColor;
-    lightText      : @lightGreyTextColor;
-    hoverText      : @greyHoverTextColor;
-    focus          : @greyFocus;
-    lightFocus     : @lightGreyFocus;
-    down           : @greyDown;
-    lightDown      : @lightGreyDown;
-    active         : @greyActive;
-    lightActive    : @lightGreyActive;
-    shadow         : @greyTextShadow;
-    lightShadow    : @lightGreyTextShadow;
-    hover          : @greyHover;
-    lightHover     : @lightGreyHover;
-    ribbon         : @greyRibbonShadow;
-    invertedRibbon : @greyInvertedRibbonShadow;
-    tertiary       : @greyTertiaryColor;
-    tertiaryHover  : @greyTertiaryColorHover;
-    tertiaryFocus  : @greyTertiaryColorFocus;
-    tertiaryActive : @greyTertiaryColorActive;
-    bright         : @greyBright;
-    brightHover    : @greyBrightHover;
-    isDark         : true;
-    isVeryDark     : false;
-  };
-  @black: {
-    color          : @black;
-    light          : @lightBlack;
-    border         : @blackBorderColor;
-    background     : @blackBackground;
-    header         : @blackHeaderColor;
-    boxShadow      : @blackBoxShadow;
-    boxFloatShadow : @blackBoxFloatingShadow;
-    text           : @blackTextColor;
-    lightText      : @lightBlackTextColor;
-    hoverText      : @blackHoverTextColor;
-    focus          : @blackFocus;
-    lightFocus     : @lightBlackFocus;
-    down           : @blackDown;
-    lightDown      : @lightBlackDown;
-    active         : @blackActive;
-    lightActive    : @lightBlackActive;
-    shadow         : @blackTextShadow;
-    lightShadow    : @lightBlackTextShadow;
-    hover          : @blackHover;
-    lightHover     : @lightBlackHover;
-    ribbon         : @blackRibbonShadow;
-    invertedRibbon : @blackInvertedRibbonShadow;
-    tertiary       : @blackTertiaryColor;
-    tertiaryHover  : @blackTertiaryColorHover;
-    tertiaryFocus  : @blackTertiaryColorFocus;
-    tertiaryActive : @blackTertiaryColorActive;
-    bright         : @blackBright;
-    brightHover    : @blackBrightHover;
-    isDark         : true;
-    isVeryDark     : true;
-  };
-};
```

Comment:
Don't remove the semicolon here, it was intentially added by #1833 to support a buggy PHP LESS mentioned in #1832

Reply:
see https://github.com/fomantic/Fomantic-UI/pull/2592#discussion_r1044866718, fixed